context-free.md

```diff
@@ -2,21 +2,55 @@
 
 ##  Who is this programming language for?
 
+Context Free is useful for people who are interested in creating
+computer-generated art.  This could have uses for graphical designs or just
+expressing creativity.  Other features in Context Free could interest animators
```

Comment:
I can also see how it might be easy to make slight tweaks to the image with just small changes to the source code, in what is probably a quicker and cheaper method than a full blown animation suite.